inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -533,6 +535,70 @@ MKLDNNConvolutionNode::createDescriptorInternal(const mkldnn::memory::desc& inpu
     return conv_desc;
 }
 
+std::vector<VectorDims> MKLDNNConvolutionNode::shapeInfer() const {
+    // pass in all input values by default
+    return const_cast<MKLDNNConvolutionNode *>(this)->shapeInferInternal({});
+}
+
+std::vector<VectorDims> MKLDNNConvolutionNode::shapeInferInternal(const std::vector<Shape>& shapes) {
+    // collect input shapes
+    std::vector<ov::StaticShape> input_shapes;
+    if (shapes.empty()) {
+        // from graph runtime
+        for (size_t i = 0; i < opToShapeInfer->get_input_size(); i++) {
+            ov::StaticShape shape(getParentEdgesAtPort(i)[0]->getMemory().getStaticDims());
+
+            // convert to scalar for length-1 1D if required by ngraph
+            if (shape.size() == 1 && shape[0] == 1 && opToShapeInfer->get_input_partial_shape(i).rank().compatible(0))
+                shape = ov::StaticShape();
+
+            input_shapes.push_back(shape);
+        }
+    } else {
+        // from parameter given
+        for (size_t i = 0; i < shapes.size(); i++)
+            input_shapes.push_back(shapes[i].getStaticDims());
+    }
+
+    std::vector<ov::StaticShape> output_shapes(opToShapeInfer->get_output_size());
+
+    shape_inference(opToShapeInfer.get(), input_shapes, output_shapes);
+
+    if (isDynamicNode() && autoPadding) {
+        ov::CoordinateDiff pads_begin, pads_end;
+        if (isGrouped) {
+            if (auto groupConvolutionOp = ov::as_type_ptr<ov::op::v1::GroupConvolution>(opToShapeInfer)) {
+                IE_ASSERT(ov::op::v1::resolve_auto_pad_for_shape(groupConvolutionOp.get(),
+                                                                 pads_begin,
+                                                                 pads_end,
+                                                                 input_shapes,
+                                                                 2,
+                                                                 3));
+                paddingL = groupConvolutionOp->get_pads_begin();
+                paddingR = groupConvolutionOp->get_pads_end();
```

Comment:
why do we take pads from the node and not the calculated ones?

Reply:
That's my fault, Thank for pointing out!